### ISSUE TITLE
Calendar keyboard accessibility

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1671]
Relevant Components

- All Calendars

Description
All controls should be available to both mouse users and keyboard users. In this case, the operation of the page is completely dependent on the user clicking with a mouse. Keyboard users are unable to select dates on a calendar through any means. The page should be 100% usable by a person without a mouse even present.

Keyboard users expect to be able to interact with focused interactive elements via the enter/return key and the spacebar.

Solution
The code will require multiple additions. Make sure the element which causes the popup is interactable via keyboard (enter/return and spacebar), as well as the popup itself. Inside the popup, make sure users can navigate with arrow keys and the tab key.

**Comment by jcreel** [comment]
Testing suggests this is nearly done.  Just need to make enter and spacebar invoke the calendar popup when the focus is on the data field.  Currently, only the down arrow will invoke the calendar popup.
